.github/workflows/stale_prs_and_issues.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Close stale issues and PRs'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-message: 'This issue will be closed in 2 weeks in case of no activity.'
```

Comment:
I suppose `in a week` ? Because `days-before-close` is 7

Reply:
right, one week should be here